source/secrets/index.rst

```diff
@@ -90,10 +90,10 @@ Dans le cas de OAuth, la clé de signature est composée de la clé secrète (ab
 
 Le processus de récupération des données à une API est le suivant:
 
-- 1 On envoie au serveur de vérification le message de requête comprenant la clé d'utilisateur. À celui-ci est ajouté le hash généré à partir du message et de la clé de signature (c'est la que :py:mod:`hmac` est utile). Cette opération est effectuée à chaque fois que le client essaye de se connecter.
-- 2 Si le destinataire du message est vérifié et que le compte est autorisé, le serveur renvoie à l'application un jeton.
-- 3 L'application envoie ce jeton au serveur de ressources qui vérifie grace à celui-ci si l'application est autorisée, et ses privilèges.
-- 4 Si l'application est autorisée, le serveur renvoie les informations requises.
+#. On envoie au serveur de vérification le message de requête comprenant la clé d'utilisateur. À celui-ci est ajouté le hash généré à partir du message et de la clé de signature (c'est la que :py:mod:`hmac` est utile). Cette opération est effectuée à chaque fois que le client essaye de se connecter.
+#. Si le destinataire du message est vérifié et que le compte est autorisé, le serveur renvoie à l'application un jeton.
+#. L'application envoie ce jeton au serveur de ressources qui vérifie grace à celui-ci si l'application est autorisée, et ses privilèges.
+#. Si l'application est autorisée, le serveur renvoie les informations requises.
```

Comment:
D'après ce que j'ai lu c'est encore plus compliqué que mon résumé..

https://www.digitalocean.com/community/tutorials/an-introduction-to-oauth-2#abstract-protocol-flow
https://help.salesforce.com/articleView?id=remoteaccess_oauth_web_server_flow.htm&language=en&type=0
http://www.bubblecode.net/fr/2016/01/22/comprendre-oauth2/#Lrsquoautorisation_via_un_code_Authorization_Code_Grant